src/scss/components/_react-modal.scss

```diff
@@ -0,0 +1,72 @@
+@import '../mixins/prefix';
+
+.#{prefix(react-modal)} {
+
+  &__overlay {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    position: fixed;
+    z-index: 10000000;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+
+    background-color: rgba(255, 255, 255, 0.75);
+  }
+
+  &__header {
+    margin-bottom: 10px;
```

Comment:
should use the existing sass mixins/functions that are grid/variable driven instead of hardcoded values. Later we will upgrade this to design tokens